projects/mercury/src/file/FileBrowser.js

```diff
@@ -107,42 +107,40 @@ export const FileBrowser = (props: FileBrowserProperties) => {
     const [currentUpload, setCurrentUpload] = useState({});
-    const [isFolderUpload, setIsFolderUpload] = useState();
 
-    const {
-        getRootProps,
-        getInputProps,
-        isDragActive,
-        isDragAccept,
-        isDragReject,
-        open
-    } = useDropzone({
-        noClick: true,
-        noKeyboard: true,
-        multiple: true,
-        onDropAccepted: (droppedFiles) => {
-            const newUpload = {
-                id: generateUuid(),
-                files: droppedFiles,
-                destinationPath: openedPath,
-            };
-            const newOverwriteFolderCandidates = getConflictingFolders(droppedFiles, existingFolderNames);
-            const newOverwriteFileCandidates = getConflictingFiles(droppedFiles, existingFileNames);
+    function InitializeDropzone() {
+        return useDropzone({
+            noClick: true,
+            noKeyboard: true,
+            multiple: true,
+            useFsAccessApi: false,
+            onDropAccepted: (droppedFiles) => {
+                const newUpload = {
+                    id: generateUuid(),
+                    files: droppedFiles,
+                    destinationPath: openedPath,
+                };
+                const newOverwriteFolderCandidates = getConflictingFolders(droppedFiles, existingFolderNames);
+                const newOverwriteFileCandidates = getConflictingFiles(droppedFiles, existingFileNames);
 
-            if (newOverwriteFileCandidates.length > 0 || newOverwriteFolderCandidates.length > 0) {
-                setOverwriteFileCandidateNames(newOverwriteFileCandidates);
-                setOverwriteFolderCandidateNames(newOverwriteFolderCandidates);
-                if (isOverwriteCandidateDeleted([...newOverwriteFileCandidates, ...newOverwriteFolderCandidates])) {
-                    setShowCannotOverwriteWarning(true);
-                    return;
+                if (newOverwriteFileCandidates.length > 0 || newOverwriteFolderCandidates.length > 0) {
+                    setOverwriteFileCandidateNames(newOverwriteFileCandidates);
+                    setOverwriteFolderCandidateNames(newOverwriteFolderCandidates);
+                    if (isOverwriteCandidateDeleted([...newOverwriteFileCandidates, ...newOverwriteFolderCandidates])) {
+                        setShowCannotOverwriteWarning(true);
+                        return;
+                    }
+                    setCurrentUpload(newUpload);
+                    setShowOverwriteConfirmation(true);
+                } else {
+                    startUpload(newUpload).then(refreshFiles);
                 }
-                setCurrentUpload(newUpload);
-                setShowOverwriteConfirmation(true);
-            } else {
-                startUpload(newUpload).then(refreshFiles);
             }
-        }
-    });
+        });
+    }
+
+    const fileDropzoneProperties = InitializeDropzone();
+    const folderDropzoneProperties = InitializeDropzone();
```

Comment:
with single `dropZone = InitializeDropzone()` instead of separate fileDropZone and folderDropZone it also seems to work correctly

Reply:
I tried that, but in the end didn't succeed because how would you handle the 2 click events? 

```
    const uploadFolder = () => {
        folderDropzoneProperties.open();
    };

    const uploadFile = () => {
        fileDropzoneProperties.open();
    };
```

Reply:
ok, you're right, I didn't notice that now there are always 2 input fields being rendered, no longer depending on the flag `isFolderUpload`